src/Chip/Remove.js

```diff
@@ -29,7 +30,7 @@ export class Remove extends PureComponent {
     render() {
         const onRemove = this.props.onRemove
 
-        if (!onRemove) return
+        if (!onRemove) return null
```

Comment:
Why `null` instead of `undefined`?

Reply:
I think it doesn't really make a difference, but interestingly I wanted to know if there's any official docs about this (I usually returned undefined so far) and found this:
https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering

So I just copied that

Reply:
Oh, that's interesting, it _does_ affect the CodePen linked: https://codepen.io/gaearon/pen/Xjoqwm?editors=0010

Try switching the `return null` to `return`. Then hit the `Hide` button. Now it's gone completely whereas if you return `null` the button is still there. React treats them differently, one stops rendering and one doesn't.

>Returning null from a component’s render method does not affect the firing of the component’s lifecycle methods. For instance `componentDidUpdate` will still be called.

Reply:
> Now it's gone completely whereas if you return null the button is still there.

Huh... That's VERY weird behavior, will never use undefined again!